src/components/panel/index.jsx

```diff
@@ -77,31 +113,36 @@ export default class Panel extends Component {
   }
   render() {
     const {
-      user: { userNickName, userEmail, isUserAuthenticated },
-      comments
+      user: { userName, userEmail, isUserAuthenticated },
+	    userComment,
+        comments
     } = this.state;
 
     return (
-          <section className="panel-container">
+      <section className="panel-container">
 
-	          { !!comments &&
-                <Comments comments={comments} />
-	          }
+        { !!comments &&
+          <Comments comments={comments} />
+        }
 
-			  { !isUserAuthenticated &&
-                <Register
-                  onUserNickNameChange={this.onUserNickNameChange}
-                  onUserEmailChange={this.onUserEmailChange}
-                  onRegisterSubmit={this.onRegisterSubmit}
-                  userNickName={userNickName}
-                  userEmail={userEmail} />
-			  }
+        { !isUserAuthenticated &&
+          <Register
+            onUserNameChange={this.onUserNameChange}
+            onUserEmailChange={this.onUserEmailChange}
+            onRegisterSubmit={this.onRegisterSubmit}
+            userName={userName}
+            userEmail={userEmail} />
+        }
 
-			  { !!isUserAuthenticated &&
-                <SubmitComment />
-			  }
+        { !!isUserAuthenticated &&
+          <SubmitComment
+            userComment={userComment}
+            onUserCommentChange={this.onUserCommentChange}
```

Comment:
Not sure what you mean, this handler just updates local state. It doesnt have any http concerns